src/dmd/astbase.d

```diff
@@ -6809,5 +6809,7 @@ struct ASTBase
     struct Target
     {
         extern (C++) __gshared int ptrsize;
+        extern (C++) __gshared bool is64bit;
```

Comment:
this field doesn't seem to be needed anywhere at parse time.

Reply:
true.